pom.xml

```diff
@@ -305,7 +311,7 @@
         <inbound.auth.oauth.version>6.5.3</inbound.auth.oauth.version>
         <commons-collections.version>3.2.0.wso2v1</commons-collections.version>
         <carbon.kernel.version>4.10.47</carbon.kernel.version>
-        <identity.framework.version>7.8.114</identity.framework.version>
+        <identity.framework.version>7.8.127</identity.framework.version>
```

Comment:
Also bump the charon version after merging the following PR: https://github.com/wso2/charon/pull/431